example/cumulus-tf/thin_egress_app.tf

```diff
@@ -63,7 +63,7 @@ resource "aws_cloudwatch_log_subscription_filter" "egress_api_gateway_log_subscr
 resource "aws_cloudwatch_log_group" "egress_lambda_log_group" {
   count             = (var.log_destination_arn != null) ? 1 : 0
   name              = "/aws/lambda/${module.thin_egress_app.egress_lambda_name}"
-  retention_in_days = 30
+  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "cumulus-tf_egress_lambda_log_retention", 30)
```

Comment:
Completely understood if it's considered out of the scope of this ticket, but since the literal default of "30" is repeated here many times, might it make sense to configure it as a variable and reference the variable?  That'll give the added bonus of being able to configure log retention values at a module level by overriding that variable.

Reply:
Correct, the lookup here would look like `lookup(var.cloudwatch_log_retention_periods, "cumulus-tf_egress_lambda_log_retention", var.default_log_retention_days)`